atmo/coordinator/capabilities/capabilities.go

```diff
@@ -2,42 +2,42 @@ package capabilities
 
 import (
 	"github.com/pkg/errors"
+
 	"github.com/suborbital/atmo/atmo/appsource"
 	"github.com/suborbital/reactr/rcap"
 	"github.com/suborbital/vektor/vlog"
 )
 
-// Render "renders" capabilities by layering any user-defined
-// capabilities onto the provided set, thus allowing the user to omit any
-// individual capability (or all of them) to receive the defaults.
-func Render(config rcap.CapabilityConfig, source appsource.AppSource, log *vlog.Logger) (rcap.CapabilityConfig, error) {
-	userConfig := source.Capabilities()
+func ResolveFromSource(source appsource.AppSource, ident, namespace, version string, log *vlog.Logger) (rcap.CapabilityConfig, error) {
```

Comment:
Looks like we lost the comment here

Reply:
Readded in https://github.com/suborbital/atmo/pull/121/commits/31130b7d653fa2e959c85153739730d0d74d38ba